partcad/src/partcad/utils_ocp.py

```
@@ -0,0 +1,111 @@
+    if angularTolerance is None:
+        angularTolerance = 0.1
+
+    if not BRepTools.Triangulation_s(shape, tolerance):
+        BRepMesh_IncrementalMesh(
+            shape,
+            theLinDeflection=tolerance,
+            isRelative=False,
+            theAngDeflection=angularTolerance,
+            isInParallel=True,
+        )
+    vertices: List[Tuple[float, float, float]] = []
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)